src/audio/mfcc/mfcc_win.c

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
These window functions would fit math library, they are generic and nearly always used with FFT.